lib/twilio/audiohelper.ts

```diff
@@ -21,6 +21,12 @@ const kindAliases: Record<string, string> = {
   audiooutput: 'Audio Output',
 };
 
+export const defaultEnabledSounds: Record<Device.ToggleableSound, boolean> = {
```

Comment:
I think we can remove these defaults from `device` if we want the `AudioHelper` class to have more control over the enablement of sounds.

Reply:
Sorry, didn't see the `device` changes, please ignore this comment! 😓